src/graphql-global-id/src/index.js

```diff
@@ -0,0 +1,8 @@
+// @flow srtict
```

Comment:
```suggestion
// @flow strict
```

We should update our Eslint rule to catch invalid values. Could you please create an issue and link this PR?